R/db-odbc-oracle.R

```diff
@@ -74,9 +74,24 @@ sql_subquery.Oracle <- function(con, from, name = unique_name(), ...) {
   }
 }
 
-# registered onLoad located in the zzz.R script
+#' @export
+db_drop_table.Oracle <- function(con, table, force = FALSE, ...) {
+  if (db_has_table(con, table) && force) {
+    sql <- build_sql(
+      "BEGIN ",
+      "EXECUTE IMMEDIATE 'DROP TABLE \"", sql(table), "\"';",
```

Comment:
Should this be `ident()` rather than `sql()`? Or do we know that it's escaped elsewhere?